src/mpi/romio/adio/ad_quobytefs/ad_quobytefs.c

```diff
@@ -0,0 +1,104 @@
+/* -*- Mode: C; c-basic-offset:4 ; indent-tabs-mode:nil ; -*- */
+/*
+ * Copyright (C) by Argonne National Laboratory
+ *     See COPYRIGHT in top-level directory
+ */
+
+
+#include "ad_quobytefs.h"
+
+#include "adioi.h"
+#include <string.h>
+
+struct ADIOI_Fns_struct ADIO_QUOBYTEFS_operations = {
+    ADIOI_QUOBYTEFS_Open,       /* Open */
+    ADIOI_GEN_OpenColl, /* OpenColl */
+    ADIOI_QUOBYTEFS_ReadContig, /* ReadContig */
+    ADIOI_QUOBYTEFS_WriteContig,        /* WriteContig */
+    ADIOI_GEN_ReadStridedColl,  /* ReadStridedColl */
+    ADIOI_GEN_WriteStridedColl, /* WriteStridedColl */
+    ADIOI_GEN_SeekIndividual,   /* SeekIndividual */
+    ADIOI_QUOBYTEFS_Fcntl,      /* Fcntl */
+    ADIOI_GEN_SetInfo,  /* SetInfo */
+    ADIOI_GEN_ReadStrided,      /* ReadStrided */
+    ADIOI_GEN_WriteStrided,     /* WriteStrided */
+    ADIOI_QUOBYTEFS_Close,      /* Close */
+#ifdef ROMIO_HAVE_WORKING_AIO
+    ADIOI_QUOBYTEFS_IreadContig,        /* IreadContig */
+    ADIOI_QUOBYTEFS_IwriteContig,       /* IwriteContig */
+#else
+    ADIOI_FAKE_IreadContig,     /* IreadContig */
+    ADIOI_FAKE_IwriteContig,    /* IwriteContig */
+#endif
+    ADIOI_GEN_IODone,   /* ReadDone */
+    ADIOI_GEN_IODone,   /* WriteDone */
+    ADIOI_GEN_IOComplete,       /* ReadComplete */
+    ADIOI_GEN_IOComplete,       /* WriteComplete */
+    ADIOI_GEN_IreadStrided,     /* IreadStrided */
+    ADIOI_GEN_IwriteStrided,    /* IwriteStrided */
+    ADIOI_QUOBYTEFS_Flush,      /* Flush */
+    ADIOI_QUOBYTEFS_Resize,     /* Resize */
+    ADIOI_QUOBYTEFS_Delete,     /* Delete */
+    ADIOI_GEN_Feature,  /* Features */
+    "QUOBYTEFS:ROMIO driver for quobyte file system",
+    ADIOI_GEN_IreadStridedColl, /* IreadStridedColl */
+    ADIOI_GEN_IwriteStridedColl,        /* IwriteStridedColl */
+    ADIOI_QUOBYTEFS_SetLock     /* SetLock */
+};
+
+static char *ADIOI_QUOBYTEFSI_ExtractRegistry(const char *filename, int *error_code)
```

Comment:
static functions don't need a namespace.  These symbols are not exported outside this file.